test/unit/saveArticles.test.ts

```diff
@@ -4,7 +4,7 @@ import domino from 'domino'
 import { setupScrapeClasses } from '../util.js'
 import { saveArticles } from '../../src/util/saveArticles.js'
 import { ZimArticle } from '@openzim/libzim'
-import { mwRetToArticleDetail, DELETED_ARTICLE_ERROR } from '../../src/util/index.js'
+import { mwRetToArticleDetail, DELETED_ARTICLE_ERROR, ARTICLE_HEADER_CLASS } from '../../src/util/index.js'
```

Comment:
Should not come from `const.js`

Reply:
Ok, I added the article header class as a regular string now.